test/unit/test-resources.js

```diff
@@ -713,6 +712,47 @@ describes.realWin('Resources discoverWork', {amp: true}, (env) => {
     expect(mediaSpy).to.be.calledOnce;
   });
 
+  describe('intersect-resources', () => {
+    beforeEach(() => {
+      // Enable "intersect-resources" experiment.
+      resources.intersectionObserver_ = {};
+      resource1.intersect_ = resource2.intersect_ = true;
+    });
+
+    it('should not applySizesAndMediaQuery after build', () => {
+      resources.relayoutAll_ = false;
+
+      // Unmeasured elements.
+      env.sandbox.stub(resource1, 'applySizesAndMediaQuery');
+      env.sandbox.stub(resource1, 'hasBeenMeasured').returns(false);
+
+      // Measured elements that need relayout.
+      env.sandbox.stub(resource2, 'applySizesAndMediaQuery');
+      env.sandbox.stub(resource2, 'hasBeenMeasured').returns(true);
```

Comment:
nit/q: is there any more reason to modify hasBeenMeasured here as there is in the relayout test case?

Reply:
`!hasBeenMeasured()` and `getState() == ResourceState.NOT_LAID_OUT` are the two conditions that are checked in the classical model that we drop in intersection mode. This specific `hasBeenMeasured` makes explicit that we're checking two distinct cases.